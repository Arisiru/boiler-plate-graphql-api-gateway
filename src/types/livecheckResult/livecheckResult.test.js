const schema = require('./schema');
const resolver = require('./resolver');

describe('[Type.LivecheckResult]', () => {
  it('should compile schema', async () => {
    expect(schema).toMatchSnapshot();
  });

  const livecheckResult = Object.freeze({
    status: 'OK',
  });

  it('maps LivecheckResult to "status" property', () => {
    expect(resolver.LivecheckResult.status(livecheckResult)).toEqual(
      livecheckResult.status
    );
  });
});
