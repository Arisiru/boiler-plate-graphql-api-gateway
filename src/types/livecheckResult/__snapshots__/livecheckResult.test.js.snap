// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[Type.LivecheckResult] should compile schema 1`] = `
Object {
  "definitions": Array [
    Object {
      "description": undefined,
      "directives": Array [],
      "fields": Array [
        Object {
          "arguments": Array [],
          "description": undefined,
          "directives": Array [],
          "kind": "FieldDefinition",
          "name": Object {
            "kind": "Name",
            "value": "status",
          },
          "type": Object {
            "kind": "NonNullType",
            "type": Object {
              "kind": "NamedType",
              "name": Object {
                "kind": "Name",
                "value": "String",
              },
            },
          },
        },
      ],
      "interfaces": Array [],
      "kind": "ObjectTypeDefinition",
      "name": Object {
        "kind": "Name",
        "value": "LivecheckResult",
      },
    },
  ],
  "kind": "Document",
  "loc": Object {
    "end": 50,
    "start": 0,
  },
}
`;
